openscm/core.py

```diff
@@ -214,17 +221,25 @@ def get_timeseries_view(
         parameter = self._get_or_create_parameter(
             name, self._get_or_create_region(region)
         )
-        timeseries = Timeseries(start_time, period_length)
-        parameter.attempt_read(unit, ParameterType.TIMESERIES, timeseries)
-        return TimeseriesView(parameter, unit, timeseries)
+        parameter.attempt_read(unit, timeseries_type, time_points)
+        return TimeseriesView(
+            parameter,
+            unit,
+            time_points,
+            timeseries_type,
+            interpolation_type,
+            extrapolation_type,
+        )
 
     def get_writable_timeseries_view(
         self,
         name: Tuple[str],
         region: Tuple[str],
         unit: str,
-        start_time: int,
-        period_length: int,
+        time_points: Tuple[int],
+        timeseries_type: ParameterType = ParameterType.AVERAGE_TIMESERIES,
```

Comment:
we should probably make the default point timeseries in future right? More intuitive?

Reply:
yes. would also remove a default here for now if that's ok?

Reply:
good for me